.github/workflows/staticmajor.yml

```diff
@@ -0,0 +1,13 @@
+name: Staticmajor
+on: [push,workflow_dispatch,pull_request]
```

Comment:
It may make sense to refine push to target only the main branch to avoid double runs:
![image](https://user-images.githubusercontent.com/6455350/190025681-67545fc2-9fdb-4575-bc5a-2fecd78d479b.png)
Eg:
```yaml
on:
  push:
    branches:
      - main
  pull_request:
  workflow_dispatch:
```

Reply:
Gotcha, thank you @kirbyquerby! Done!